job-view.php

```diff
@@ -16,20 +8,31 @@
 if (mysqli_num_rows($result) < 1) {
     header('Location: ' . BASE_URL);
 }
+$row = mysqli_fetch_assoc($result);
+
+
+//params for include template
+$meta_title = $row['title'] . " | Jobrix.tk";
 
-while ($row = mysqli_fetch_assoc($result)) :
-    ?>
-    <section class="section-fullwidth">
-        <div class="row">
-            <div class="job-single">
-                <div class="job-main">
-                    <div class="job-card">
-                        <div class="job-primary">
-                            <header class="job-header">
-                                <h2 class="job-title"><?php echo $row['title'] ?></h2>
-                                <div class="job-meta">
-                                    <a class="meta-company" href="#"><?php echo $row['company_name'] ?></a>
-                                    <span class="meta-date">
+//header template include
+require_once(dirname(__FILE__) . '/includes/theme-compat/header.php');
+
+//Checks if a number is passed as a parameter in the _get request and if the given ID exists in the database
+$jobs_id = "";
+if (!isset($_GET['id']) || !is_numeric($_GET['id'])) {
+    header('Location: ' . BASE_URL);
+} ?>
+<section class="section-fullwidth">
+    <div class="row">
+        <div class="job-single">
+            <div class="job-main">
+                <div class="job-card">
+                    <div class="job-primary">
+                        <header class="job-header">
+                            <h2 class="job-title"><?php echo $row['title'] ?></h2>
+                            <div class="job-meta">
+                                <a class="meta-company" href="#"><?php echo $row['company_name'] ?></a>
```

Comment:
the link here is not going to work?

Reply:
There is no suitable HTML page to link to. There is no suitable HTML template. I'm not sure exactly how to solve the problem with this empty link, so I've left it empty.

Reply:
just check if `$row['company_name'] ` is empty - if yes, do no display the `<a>` element

Reply:
Fixed: For the given companies without a website, the name will not be displayed as a link.

```
if ( ! empty( $job['company_site'] ) ) { ?>
    <a class="meta-company" href="<?php echo 'http://' . $job['company_site'] ?>"><?php echo $job['company_name'] ?></a>
<?php } else {
	echo $job['company_name'];
}

```